### ISSUE TITLE
Carbon v9 Dropdown shows accessibility issue with DAP tool 

### ISSUE THREAD

**Issue by SowmyaRaviKrishnan** [carbon-design-system/carbon#4643]
Carbon v9 Dropdown shows accessibility issue with DAP tool 
1.An interactive element/widget must have an accessible name.
## Environment

> Operating system
Linux - RHEL
> Browser
Chrome
> Assistive technology used to verify
IBM DAP
## Detailed description

> What version of the Carbon Design System are you using?
v9
> What WCAG 2.1 checkpoint does the issue violate?
IBM 2.5.3 Label in Name

**Comment by SowmyaRaviKrishnan** [comment]
@joshblack @amuqeetkhan

**Comment by abbeyhrt** [comment]
Hi @SowmyaRaviKrishnan, thanks for bringing this to our attention! As I've mentioned in the other issues for v9, we will not be backporting WCAG 2.1 fixes to v9 but we always accept community contributions for any v9 issues! If you would like this to be addressed, feel free to open up a PR and we’ll review it! 👍
